docs/whitelabelling.md

```diff
@@ -0,0 +1,105 @@
+# Whitelabelling
```

Comment:
Not sure if we use British or American English, but if American "Labeling" would only contain one "L".